### PR TITLE
html-syntax - Simon Kim.

### PR DESCRIPTION
**Pull request by Skim-329**
Screen Shot 2021-08-16 at 1 27 06 PM

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<body>
```
Comment (Zhiwutian) [inline html-syntax/index.html:2]: Hey Simon, make sure to include the rest of the HTML skeleton components here, such as the html and head elements :)

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: html-syntax/index.html (hunk 1 above)
Comment (Skim-329) [reply html-syntax/index.html:2]: Screen Shot 2021-08-16 at 2 43 52 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**